tests/integration/datadict/submission/test_admin_endpoints.py

```diff
@@ -33,16 +33,27 @@ def post_blgsp_files(client, headers):
 
     test_fnames = data_fnames + ["read_group.json", "submitted_unaligned_reads.json"]
 
-    entity_types = [fname.replace(".json", "") for fname in data_fnames]
+    # entity_types = [fname.replace(".json", "") for fname in data_fnames]
+    entity_types = {}
+    for fname in data_fnames:
+        fname = fname.split(".")[0]
+        if fname not in entity_types:
+            entity_types[fname] = 1
+        else:
+            entity_types[fname] = entity_types.get(fname) + 1
     resp = post_example_entities_together(client, headers, data_fnames2=test_fnames)
     assert resp.status_code == 201, resp.data
 
-    submitted_entities = {
-        entity["type"]: entity["id"] for entity in resp.json["entities"]
-    }
+    submitted_entities = {}
+    for entity in resp.json["entities"]:
+        if entity["type"] not in submitted_entities:
+            submitted_entities[entity["type"]] = [entity["id"]]
+        else:
+            submitted_entities.get(entity["type"]).append(entity["id"])
```

Comment:
you don't have to, but you could use defaultdict here
```
from collections import defaultdict
submitted_entities = defaultdict(list)
for entity in resp.json["entities"]:
    submitted_entities[entity["type"]].append(entity["id"])
```
and `defaultdict(int)` would work for `entity_types` above too